app/settings/account/-components/opt-out/component.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import CurrentUserService from 'ember-osf-web/services/current-user';
+import { restartableTask } from 'ember-concurrency';
+import { waitFor } from '@ember/test-waiters';
+import config from 'ember-get-config';
+import IntlService from 'ember-intl/services/intl';
+import captureException, { getApiErrorMessage } from 'ember-osf-web/utils/capture-exception';
+
+const { support: { supportEmail } } = config;
+
+export default class OptOutComponent extends Component {
+    @service currentUser!: CurrentUserService;
+    @service intl!: IntlService;
+    @service toast!: Toastr;
+
+    get allowIndexingIsFalse() {
+        // allowIndexing is null by default
```

Comment:
What a pain that is.

Reply:
Yea, `allow_indexing: None` is the default and that indexes users. Ideally this panel would just be a checkbox, but alas